A whole new UI and fixed missing default icons for iPad.

ipodtouchdude opened the pull request:
Changed the whole UI again.
Fixed missing default icons for iPads.

![](https://i.imgur.com/WL5EPv3.png)
![](https://i.imgur.com/29Xejcv.png)
![](https://i.imgur.com/Jk0ybux.png)
![](https://i.imgur.com/lJMsCbE.png)

ZonD80 commented:
You need to fix:

- Bug with layout on iPhone X
- Ability to get DRM-free version from appdb
![IMG_4764B2597CCF-1](https://user-images.githubusercontent.com/273057/84173941-a0e62b80-aa86-11ea-94da-bf71a45b86cb.jpeg)

ipodtouchdude commented:
Fixed the UI bugs with smaller devices.
App grid now shows 4 per row on iPhone and 6 per row on iPad.
Changed the Duplicate view from a UILabel (app name) to a UINavigationBar for more space on smaller devices.
Fixed square icons on iPad Duplicate view.

![](https://i.imgur.com/YHyaY19.png)
![](https://i.imgur.com/8ztoQer.png)
![](https://i.imgur.com/UhdsfvK.png)
![](https://i.imgur.com/Q03eYtN.png)
